01-intro.Rmd

```diff
@@ -37,24 +59,82 @@ The major requirement of the policy is that all grant proposals (submitted after
 
 For certain grant mechanisms for projects that do not generate data, compliance with the policy is not required. For certain types of data, sharing is not possible, and a justification will be required instead. 
 
-What about grant renewals? 
+The following outline several key questions:
+
+- **Does my research generate scientific data?** If yes, you will need to submit a DMS Plan.
```

Comment:
I'm still unclear if this is true for some mechanisms

Reply:
going off of their slide here. What do you think?
![image](https://user-images.githubusercontent.com/15618412/197617934-c1ea7d08-845b-47cf-824f-4fe88ca17e26.png)

Reply:
maybe "follow the policy?"

Reply:
yeah they said they would provide info about activity codes - hopeful that link will or already exists and we can just point them to that